src/App.js

```diff
@@ -2,14 +2,34 @@ import React from "react";
 import { Switch, Redirect, Route, withRouter } from "react-router-dom";
 import AuthenticationRoute from "./containers/AuthenticationRoute";
 import routes from "./helpers/routes";
+import { DEFAULT_CURRENCY } from "./config";
 import LoginScreen from "./components/LoginScreen";
+import Currency from "./components/Currency";
+
 
 const App = props => {
-  const isAuthenticated = true;
+  const isAuthenticated = false;
+  const currencyValue = 1500015646;
+  let currency = "";
+  currency = currency || DEFAULT_CURRENCY;
```

Comment:
This needs to be done in the component otherwise we will need to do this every-time.